ml-agents/mlagents/trainers/trajectory.py

```diff
@@ -8,8 +8,8 @@ class AgentExperience(NamedTuple):
     obs: List[np.ndarray]
     reward: float
     done: bool
-    action: np.ndarray
-    action_probs: np.ndarray
+    action: Dict[str, np.ndarray]
```

Comment:
Can these be the same datatype of the action? E.g. `ActionTuple`

Reply:
https://github.com/Unity-Technologies/ml-agents/pull/4651